generator/src/main/java/com/scottlogic/deg/generator/guice/FormatterProvider.java

```diff
@@ -0,0 +1,34 @@
+package com.scottlogic.deg.generator.guice;
+
+import com.google.inject.Inject;
+import com.google.inject.Provider;
+import com.scottlogic.deg.generator.generation.GenerationConfigSource;
+import com.scottlogic.deg.generator.outputs.datasetwriters.CsvFormatter;
+import com.scottlogic.deg.generator.outputs.datasetwriters.JsonFormatter;
+import com.scottlogic.deg.generator.outputs.datasetwriters.RowOutputFormatter;
+
+public class FormatterProvider implements Provider<RowOutputFormatter> {
+
```

Comment:
Is the empty-initial-line thing conventional within the project? I've seen it in a few places, but haven't seen anything that explicitly recommends it.

Reply:
Don't know, not something I have considered